### PR TITLE
DDF-3427 Aligned org.ops4j.pax.logging.cfg and log4j2.config.xml configurations

### PR DESCRIPTION
**Pull request by emmberk**
#### What does this PR do?
This PR fixed some discrepancies between `org.ops4j.pax.logging.cfg` and `log4j2.config.xml`.
#### Who is reviewing it? 
@gordocanchola @peterhuffer 
#### Choose 2 committers to review/merge the PR. 
@bdeining
@clockard
#### How should this be tested?

- Confirm that both log configuration files include the same suppressions.
- Confirm that logging still works when DDF is running.
- Follow the Documentation (`Enabling Fallback Audit Logging`) to switch to use the `log4j2.config.xml`, and confirm that logging still works when DDF is running.

#### Any background context you want to provide?
Is is a required hardening step to use the `log4j2.config.xml` configuration file, so it should match `org.ops4j.pax.logging.cfg`.
#### What are the relevant tickets?
[DDF-3427](https://codice.atlassian.net/browse/DDF-3427)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
For the general case, merging is gated on the following:
- [ ] Review by at least two contributors
- [ ] Review by at least two committers
- [ ] Successful CI
- [ ] Successful static analysis (SonarQube/Coverity/etc.)
- [ ] Heroing of change (details vary by ticket)

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/ddf-common/src/main/resources-filtered/etc/system.properties [hunk 1]
```diff
@@ -68,6 +63,7 @@ org.codice.ddf.system.rootContext=/services
 #
 # System Information Properties
 #
+org.codice.ddf.system.branding=${branding-lowercase}
```
Comment (clockard) [inline distribution/ddf-common/src/main/resources-filtered/etc/system.properties]: I think you commented on this in a different pr but should this property name be more generic so it makes more sense for downstream projects?

File: distribution/docs/src/main/resources/content/_configuring/auditing.adoc [hunk 2]
```diff
@@ -71,3 +71,9 @@ In the event the system is unable to write to the `security.log` file, ${brandin
 If the system is unable to write to the `security.log` file on system startup, fallback logging will be unavailable.
 Verify that the `security.log` file is properly configured and contains logs before configuring a fall back.
 ====
+
+[WARNING]
+====
```
Comment (clockard) [inline distribution/docs/src/main/resources/content/_configuring/auditing.adoc]: Thanks for adding this!

File: distribution/ddf-common/src/main/resources-filtered/etc/system.properties (hunk 1 above)
Comment (emmberk) [reply distribution/ddf-common/src/main/resources-filtered/etc/system.properties]: Yep, I did comment that, but I like @gordocanchola's idea to keep this property matching the convention of the other system properties, including `org.codice.ddf.system.siteName` which is project specific.

File: distribution/ddf-common/src/main/resources/etc/log4j2.config.xml [hunk 3]
```diff
@@ -138,15 +138,29 @@
...
+        logging.-->
+        <Logger name="graphql.execution" level="fatal"/>
         <Logger name="lux.solr" level="warn"/>
-        <Logger name="org.ops4j.pax.web.jsp" level="warn"/>
         <Logger name="org.apache.aries.spifly" level="warn"/>
+        <Logger name="org.apache.cxf" level="warn"/>
         <Logger name="org.apache.cxf.jaxrs.impl.WebApplicationExceptionMapper" level="error"/>
         <Logger name="org.apache.cxf.phase.PhaseInterceptorChain" level="error"/>
-        <!-- Camel - Suppresses an NPE warning that is ignored by Camel -->
+        <!--Suppress an NPE warning in the FileSystemStorageProvider that is ignored by Camel.-->
```
Comment (peterhuffer) [inline distribution/ddf-common/src/main/resources/etc/log4j2.config.xml]: Why was this changed? The NPE happens in Camel still.

File: distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg [hunk 4]
```diff
@@ -84,31 +84,37 @@ log4j2.logger.org_apache_lucene.appenderRef.solr.ref = solr
...
 log4j2.logger.lux_solr.level = WARN
+log4j2.logger.org_apache_aries_spifly.name = org.apache.aries.spifly
+log4j2.logger.org_apache_aries_spifly.level = WARN
+log4j2.logger.org_apache_cxf.name = org.apache.cxf
+log4j2.logger.org_apache_cxf.level = WARN
+log4j2.logger.org_apache_cxf_jaxrs_impl_webapplicationexceptionmapper.name = org.apache.cxf.jaxrs.impl.WebApplicationExceptionMapper
+log4j2.logger.org_apache_cxf_jaxrs_impl_webapplicationexceptionmapper.level = ERROR
+log4j2.logger.org_apache_cxf_phase_phaseinterceptorchain.name = org.apache.cxf.phase.PhaseInterceptorChain
+log4j2.logger.org_apache_cxf_phase_phaseinterceptorchain.level = ERROR
+# Suppress an NPE warning in the FileSystemStorageProvider that is ignored by Camel.
```
Comment (peterhuffer) [inline distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg]: Same question.

File: distribution/ddf-common/src/main/resources/etc/log4j2.config.xml (hunk 3 above)
Comment (emmberk) [reply distribution/ddf-common/src/main/resources/etc/log4j2.config.xml]: I didn't change the suppression. Do you mean why did I update the comment from `Suppresses an NPE warning that is ignored by Camel`? I'd like to give a little more context about when that error happens. Is it just a FileSystemStorageProvider thing? https://github.com/codice/ddf/pull/2604

Comment (peterhuffer) [reply distribution/ddf-common/src/main/resources/etc/log4j2.config.xml]: `<!--Suppress an NPE warning in the DefaultUnitOfWork that is ignored by Camel.-->` Maybe I shouldn't have put `that is ignored by Camel`. I'm okay with removing that part. Camel says it is ignoring the NPE in the message it prints out.

Comment (peterhuffer) [reply distribution/ddf-common/src/main/resources/etc/log4j2.config.xml]: The NPE is not coming from the `FileSystemStorageProvider` though - it is because we pass a `null` as the file through the Camel exchange on a delete of a WebDav entry.

Comment (emmberk) [reply distribution/ddf-common/src/main/resources/etc/log4j2.config.xml]: How about 
```xml
<!--Suppress a NPE that can be disregarded that occurs when a null file is passed through the Camel exchange on a delete event of a WebDav entry.-->
```
?

Comment (emmberk) [reply distribution/ddf-common/src/main/resources/etc/log4j2.config.xml]: Fixed:+1:

**Comment by clockard** [comment]
Manually ran the logging tests. Looks good :shipit:

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
build now
